app/boards/arm/dao/dao_left_defconfig

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: MIT
+
+CONFIG_ZMK_SPLIT_ROLE_CENTRAL=y
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_DAO_LEFT=y
+CONFIG_ZMK_SPLIT=y
+CONFIG_ZMK_SLEEP=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# Enable writing to flash
+CONFIG_USE_DT_CODE_PARTITION=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+CONFIG_NVS=y
+CONFIG_SETTINGS_NVS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+
+# Enable external crystal
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_XTAL=y
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_30PPM=y
+
+# # Turn on logging, and set ZMK logging to debug output
+# CONFIG_ZMK_USB_LOGGING=y
+
+# # Turn on logging, and set ZMK logging to debug output
+# CONFIG_LOG=y
+# CONFIG_ZMK_LOG_LEVEL_DBG=y
+
+# Turn on uf2 build
+CONFIG_BUILD_OUTPUT_UF2=y
+
+# Increase transmit power level
+CONFIG_BT_CTLR_TX_PWR_PLUS_8=y
```

Comment:
This is usually not set by default at the board level; you can add it to `dao.conf` file commented out if you want users to be able to enable it easily.

Reply:
Moved to `dao.conf`.